react/features/mobile/external-api/middleware.js

```diff
@@ -115,6 +125,32 @@ function _getSymbolDescription(symbol: Symbol) {
     return description;
 }
 
+/**
+ * If {@link SET_ROOM} action happens for a valid conference room this method
+ * will emit an early {@link CONFERENCE_WILL_JOIN} event to let the external API
+ * know that a conference is being joined. Before that happens a connection must
+ * be created and only then base/conference feature would emit
+ * {@link CONFERENCE_WILL_JOIN}. That is fine for the Jitsi Meet app, because
+ * that's the a conference instance gets created, but it's too late for
+ * the external API to learn that. The latter {@link CONFERENCE_WILL_JOIN} is
+ * swallowed in {@link _swallowEvent}.
+ *
+ * @param {Store} store - The redux store.
+ * @param {Action} action - The redux action.
+ * @returns {void}
+ */
+function _maybeTriggerEarlyConferenceWillJoin(store, action) {
+    const { locationURL } = store.getState()['features/base/connection'];
+    const { room } = action;
+
+    room && locationURL && _sendEvent(
```

Comment:
Don't you want to use the isRoomValid function to properly determine if the room is valid? Or a true-like value is enough for you here?

Reply:
I will use isRoomValid thanks !